packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -236,3 +238,11 @@ export function getViemNetwork(networkName: EVMBlockchainIncludingTestnet) {
             throw new Error(`Unsupported network: ${networkName}`);
```

Comment:
If we will use only `viem`, then will we lose support for `astar-zkevm`?

Reply:
If so, I propose:
- Refactor `getOrCreateWallet<B extends EVMBlockchainIncludingTestnet = EVMBlockchainIncludingTestnet>` to a class that can receive a type of actually supported chains
- Put a check at the top of the function, asserting that

Reply:
Added